fe/fe-core/src/main/java/com/starrocks/persist/EditLog.java

```diff
@@ -1035,6 +1043,10 @@ public void logAddPartition(PartitionPersistInfo info) {
         logEdit(OperationType.OP_ADD_PARTITION, info);
     }
 
+    public void logAddPartition(PartitionPersistInfoV2 info) {
```

Comment:
New log type should only be used in new features, because if fe rollback, the new log type will be ignored. But you use this log to persist rang partition and list partition, the added rang partition will be lost if fe is rollback.

Reply:
thanks for review 
1. this method logAddPartition now only use for list partition .  Look at the method called by the upper layer
2. RangePartitionPersistInfo is used as an opening , waiting for the next PR to be compatible with the old PartitionPersistInfo. So it does not compatible with the old log type nw